src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "./Introduce.scss";
+
+function Introduce() {
+  function handleInput() {
+    console.log("input 태그 동작");
+  }
+  let navigate = useNavigate();
```

Comment:
```suggestion
  const navigate = useNavigate();
```
let으로 선언하면 data의 타입이 변경될 가능성이 있기 때문에 const로 선언해서 사용해주세요!